src/app/models/affiliation.ts

```diff
@@ -3,10 +3,7 @@ import { v4 as uuidv4 } from 'uuid';
 export interface Affiliation {
   readonly id: string;
   label: string;
-  institution: {
-    name: string;
-    department: string;
-  };
+  institution: string;
```

Comment:
Longer term, and thinking about the flexibility of the tool I can only see us unwinding this change. I'd thought that we should keep the model as aligned to the JATS spec as possible and try and handle the eLife specifics in the presentation layer instead, switched on some form of config setting to adjust the behaviour. I get that is out of scope for this issue, but might be worth considering the costs/debt that we might be accumulating by not tackling these things now. Thoughts?